extension/deployment/src/main/java/io/quarkiverse/temporal/deployment/TemporalProcessor.java

```diff
@@ -348,6 +349,24 @@ void produceWorkflowStubSyntheticBeans(
 
         for (WorkflowBuildItem workflowBuildItem : workflowImplBuildItems) {
             if (workflowBuildItem.workers.length == 1) {
+
+                producer.produce(
+                        SyntheticBeanBuildItem.configure(
+                                ParameterizedType.create(TemporalInstance.class, ClassType.create(workflowBuildItem.workflow))
```

Comment:
You need to add the parameterized type to the set of bean types, e.g. with `BeanConfiguratorBase#addType(Type)`. The `configure()` method only accepts the implementation class, i.e. something like this should work:

```java
 SyntheticBeanBuildItem.configure(TemporalInstance.class)
                                .addType(ParameterizedType.create(TemporalInstance.class, ClassType.create(workflowBuildItem.workflow))
                               .scope(...)
```

Reply:
indeed, it's working now, thanks a lot !

Reply:
You're welcome! :+1: